### PR TITLE
proposed Italian translation of Home and Getting Started

### PR DESCRIPTION
**Pull request by lucab85**
I want to propose me as an Italian translator for your website. I'm Italian mother-language with tech background (B.S. in Computer Science). 
Please accept this initial contribution in Web Fundamentals Homepage and Getting started pages.

### CODE REVIEW

File: src/content/it/_common-links.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% comment %}
```
Comment (petele) [inline src/content/it/_common-links.md]: Please remove this file.

File: src/content/it/fundamentals/getting-started/_toc.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+toc:
```
Comment (petele) [inline src/content/it/fundamentals/getting-started/_toc.yaml]: We do not translate the TOC files, if we do, then any new content added will not show up in the translated version unless is explicitly added.  Please remove any `_toc.yaml` or `_book.yaml` files.

File: src/content/it/fundamentals/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+Questi codelab forniscono un'esperienza guidata, tutorial, hands-on di 
+programmazione. La maggior parte dei codelabs ti seguiranno attraverso 
+il processo di costruzione di una piccola applicazione,
+oppure di aggiunta una nuova funzionalità ad un'applicazione esistente. 
+Trova più codelabs
+[codelabs.developers.google.com](https://codelabs.developers.google.com/?cat=Web)
+
+<div class="attempt-left">
+  <a href="codelabs/your-first-pwapp/">
+    La tua prima applicazione Web Progressiva
```
Comment (petele) [inline src/content/it/fundamentals/getting-started/index.md]: we typically capitalize Application, so `applicazione` should be capitalized.

**Comment by petele** [comment]
Hey @lucab85 - sorry this is taking so long. Before we can merge, I need to get a someone who speaks Italian to verify and proof it, and I haven't been able to line anyone up yet.

**Comment by lucab85** [comment]
I perfectly understand your company policy but, if is not a problem for you, I'll continue to give my contribution.

**Comment by lucab85** [comment]
Deprecated PR

**Pull request closed without merging**